### PR TITLE
Update PPP

### LINKED ISSUE #279: PPP not finalized (OVReader)

### PR DESCRIPTION
**Pull request by OVReader**
Fix #279 
1. Finalized PPP.
2. Add some details to Target User Profile and Value Proposition in DG to add value to project.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -492,11 +492,12 @@ ___
...
 ### Target user profile
 Property agents who: 
-* are managing single owner rental units
+* are managing single owner rental units.
 * has a need to keep track of information of properties that are being put out for rental.
 * has a need to keep track of information of clients' (prospective tenants) information.
-* is a fast typist
+* is a fast typist.
 * favors a command-line interface over a Graphic User Interface.
+* work in Singapore and deal with Singapore properties.
```
Comment (wilsonngja) [inline docs/DeveloperGuide.md:500]: Good job with the constraint

File: docs/team/ovreader.md [hunk 2]
```diff
@@ -34,16 +34,18 @@ Given below are my contributions to the project.
...
 
 - **Enhancement to existing features**:
-    - Added testing for add features (To be implemented)
+    - Added testing for add features
 
 
 - **Documentation**:
     - User Guide:
-      * Added comprehensive guide for features `add -client` and `add -property` [#134](https://github.com/AY2223S1-CS2113-F11-1/tp/pull/134)
+      * Added comprehensive guide for features `add -client` and `add -property` [#134](https://github.com/AY2223S1-CS2113-F11-1/tp/pull/134), [#258](https://github.com/AY2223S1-CS2113-F11-1/tp/pull/258)
```
Comment (wilsonngja) [inline docs/team/ovreader.md:42]: Good job including the recent PRs

**Review by wilsonngja — COMMENTED** [review]
LGTM

**Review by FeliciaBeatrice — APPROVED** [review]
LGTM

**Pull request merged**